revolv/project/tests/test_models.py

```diff
@@ -11,6 +12,31 @@
 from revolv.project.models import Category, Project
 from revolv.project.tasks import scrape
 
+class ProjectUpdateTest(TestCase):
+    """Tests that check that project updates work with projects"""
+
+    def test_construct(self):
+        project = Project.factories.base.create()
+        update1 = ProjectUpdate(update_text = 'This is update text', project=project)
+        update2 = ProjectUpdate(update_text = 'This is another update', project=project)
+
+        # tests basic construction
+        self.assertEqual('This is update text', update1.update_text)
+        self.assertEqual('This is another update', update2.update_text)
+
+        # tests project relationship
+        self.assertEqual(update1.project_id, update2.project_id)
```

Comment:
yeah update1.project_id returns the same result as update1.project_pk or update1.project.pk

i think it exists so you could give things custom ids??? so not stuff relevant to this

but will change